scripts/variantstore/tieout/add_max_as_vqslod.py

```diff
@@ -19,33 +19,30 @@
 
         parts = line.split("\t")
         
-        if ("ExcessHet" in parts[6]):
+        # strip out hard filtered sites, so vcfeval can use "all-records" to plot the ROC curves
+        if ("ExcessHet" in parts[6] or "LowQual" in parts[6] or "NO_HQ_GENOTYPES" in parts[6]):
             continue
 
-        # if the "loose" argument is specified, we remove the non Excess Het site-specific filers
-        # of EXCESS_ALLELES and NO_HQ_GENOTYPES.  This is used to measure the effect of these filters
-        # when comparing to WARP (which doesn't employ these filters).
-        if (len(sys.argv) > 2 and sys.argv[2] == "loose"):
-            # undo 
-            x = parts[6].replace("EXCESS_ALLELES","").replace("NO_HQ_GENOTYPES","").strip(";")
-            if x == "":
-                parts[6] = "PASS"
-            else:
-                parts[6] = x
-        else:
-            if ("EXCESS_ALLELES" in parts[6] or "NO_HQ_GENOTYPES" in parts[6]):
-                continue;
-            
         info = parts[7]    
         d = dict([ tuple(x.split("=")) for x in info.split(";") if "=" in x]) 
 
         format_key = [x for x in parts[8].split(":")]
         sample_data = dict(zip(format_key, parts[9].split(":")))
 
         gt = sample_data['GT']
-        
+
         if gt == "0/0" or gt == "./.":
             continue;
+
+        # if there is an FT tag on the genotype, move it to the FILTER field (since this is single sample)
+        if 'FT' in sample_data:
+            ft = sample_data['FT']
+
+            if ft != "PASS" or ft != ".":
+                if (parts[6] == "PASS" or parts[6] == "."):
+                    parts[6] = ft
+                else:
+                    parts[6] = parts[6] + "," + ft
```

Comment:
this will always be true:
`if ft != "PASS" or ft != ".":`
if think you want `if ft != "PASS" and ft != ".":`

Reply:
great catch!

Reply:
reworked this, and re-ran to make it didn't change the tieout results (which it did not)